Docker.DotNet/Models/ContainerPathStats.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace Docker.DotNet.Models
+{
+    [DataContract]
+    public class ContainerPathStats
```

Comment:
Follow up to my stats vs stat comment, we can probably call this ContainerPathInfo or ContainerPathStat.